pywbem/_subscription_manager.py

```diff
@@ -389,14 +389,25 @@ def add_server(self, server):
         # Recover owned destination, filter, and subscription instances
         # that exist on the WBEMServer
         dest_name_pattern = re.compile(
+            _format(r'^pywbemdestination:{0}:[^:]*$',
+                    self._subscription_manager_id))
+        dest_name_old_pattern = re.compile(
             _format(r'^pywbemdestination:owned:{0}:{1}:[^:]*$',
                     this_client, self._subscription_manager_id))
+
         dest_insts = server.conn.EnumerateInstances(
             DESTINATION_CLASSNAME, namespace=server.interop_ns)
 
         for inst in dest_insts:
             if re.match(dest_name_pattern, inst.path.keybindings['Name']):
                 self._owned_destinations[server_id].append(inst)
+            if re.match(dest_name_old_pattern, inst.path.keybindings['Name']):
+                warnings.warn(
+                    _format(
+                        "Ignored owned listener destination instance in WBEM "
```

Comment:
I have one possible issue with this message (applies to the filter also).  The filter is not completely ignored, it appears in the list of all destinations.  The message reads as if is completely ignored.

Reply:
DONE - Changed to:
```
                        "Ignored listener destination instance when detecting "
                        "owned instances because it has the old name format: "
                        "{0}",
```